benchmarks/startup/config.yaml

```diff
@@ -0,0 +1,32 @@
+startup_with_ddtrace_run: &defaults
+  use_ddtrace_run: "true"
+  telemetry_enabled: "false"
+  runtime_metrics_enabled: "false"
+  with_trace: "false"
+
+startup_without_ddtrace_run:
```

Comment:
This would allow us to measure the overhead of importing the library. I'm not sure of the value of having this in a "startup" benchmark as there would be additional steps that would be usually taken if not using ddtrace-run, e.g, patching and configuring the tracer.

Reply:
I renamed this benchmark to `startup_and_import_ddtrace`. This should just measure the baseline of running a subprocess and importing ddtrace